jenkins/ci_cross_build.py

```diff
@@ -198,6 +198,10 @@ def check_sysroot(name: str):
     shutil.copy2(workspace_path / 'product-texts' / 'mobile' / 'couchbase-lite' / 'license' / f'LICENSE_{args.edition}.txt',
         f'libcblite-{args.version}/LICENSE.txt')
 
+    #notices.txt is produced by blackduck.
+    #It is not part of source tar, it is download to the workspace by a separate curl command by jenkins job.
+    if os.path.isfile(workspace_path / 'notices.txt'):
```

Comment:
Is this script used by production builds, or only CI/CV? is it necessary to include notices.txt for CI builds? I guess it's harmless if the job is in fact curling the file into the workspace..